1.20 release notes

codyjlandstrom opened the pull request.

ifbyol commented on src/content/release-notes.mdx:
```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
+### New Features
+
+- Add support for Openshift: Openshift Routes now show as Okteto Endpoints  <!-- 8025 -->
+- Enable Pull Secrets by default  <!-- 8001 -->
```
Should we link to [here](https://www.okteto.com/docs/admin/registry-credentials/#pull-secrets-vs-kubelet-reconfiguration)?

ifbyol replied:
Also, I just remember that this is a new way of managing credentials to pull pods in the cluster and some customers already manage that in their own. We should include a note here or in another place (maybe adding a `before upgrade to 1.20.x` [here](https://www.okteto.com/docs/self-hosted/manage/upgrade/)) to mention that if someone doesn't want Okteto to manage them, they should disabled 2 settings:
* `regcredsManager.pullSecrets.enabled` should be set to `false`
* `daemonset.configurePrivateRegistriesInNodes.enabled` should be set to `false` (if they were already preventing Okteto to manage credentials in the node, they would probably have this one already set)

ifbyol commented on src/content/release-notes.mdx:
```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
+### Improvements
+
+- Namespaces without an owner will now be picked up by the Garbage Collector  <!-- 8018 -->
+- Added the ability to assign a PriorityClass to Okteto components  <!-- 7966 -->
```
Should we indicate how his can be done? (through helm settings)

ifbyol replied:
Should we link to the property in the helm chart `globals.priorityClassName`, then? Should we also mention how do they work (preference) with the individual priorityClass you can define for Okteto components?

codyjlandstrom replied:
What section would we link to here?

ifbyol replied:
This would be the link: https://www.okteto.com/docs/1.20/self-hosted/helm-configuration/#globals

ifbyol commented on src/content/release-notes.mdx:
```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
+- Namespaces without an owner will now be picked up by the Garbage Collector  <!-- 8018 -->
+- Added the ability to assign a PriorityClass to Okteto components  <!-- 7966 -->
+- Added custom tolerations to the Helm Chart and standardize their helm values  <!-- 8029 -->
+- Fixed an issue with Custom Tolerations: The Registry will no longer inherit Buildkit tolerations  <!-- 8073 -->
```
I think this one was not a real issue as it was to fix an issue generated by the previous point of improvements, so I think it can be discarded

ifbyol reviewed (commented):
It seems that the build has failed in netlify, but not sure why

pchico83 commented on src/content/release-notes.mdx:
```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
+
+- Namespaces without an owner will now be picked up by the Garbage Collector  <!-- 8018 -->
+- Added the ability to assign a PriorityClass to Okteto components  <!-- 7966 -->
+- Added custom tolerations to the Helm Chart and standardize their helm values  <!-- 8029 -->
```
We are adding `globals` tolerations that apply to any Okteto component (pods in the okteto namespace)

ifbyol commented on src/content/release-notes.mdx:
```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
+- Included license issue warning badges on Admin and sidebar menu items  <!-- 8074 -->
+- Made error handling and feedback messages more consistent and actionable throughout the UI  <!-- 7952 -->
+- In the UI, we renamed all `/cluster/*` routes to `/admin/*`  <!-- 8002 -->
+- Updated CLI version in the backend to 2.27.1  <!-- 8089 -->
```
* We might want to mention that we have changed the heuristic to mask variables in the logs and comment that only words longer that 5 chars are masked
* Support of variables expansion on `manifest.build.dockerfile`
* Support variable expansion in variables defined in `.env`. Example `"VAR1=${VAR2:-default}"`
* Announce deprecation of compose file detection by name. (distinction between stack and compose files)

I think these are the most relevant things included in CLI

codyjlandstrom replied:
Should we create a CLI section for this?

ifbyol replied:
We have to find a better way probably, but we are usually doing something similar to this: https://www.okteto.com/docs/release-notes/#improvements

Prefixing it with `Okteto CLI`

pchico83 commented:
> @pchico83 Should we also mention that we are removing TLS validation for ingresses? There are other use cases apart from Openshift, right?

I was just thinking about it. I would mention it in a little abstract way. I mean, we didnt't document which validations we were doing. But some customers , like Spot or IDF, are waiting for that change. I would mention that "we have removed ingres tls validation to be more flexible to deploy your own ingress objects", or something in that line.

codyjlandstrom commented:
> > @pchico83 Should we also mention that we are removing TLS validation for ingresses? There are other use cases apart from Openshift, right?
> 
> I was just thinking about it. I would mention it in a little abstract way. I mean, we didnt't document which validations we were doing. But some customers are waiting for that change. I would mention that "we have removed ingres tls validation to be more flexible to deploy your own ingress objects", or something in that line.

I had removed the line item about this. Added back in

ifbyol commented on versioned_docs/version-1.20/release-notes.mdx:
```diff
@@ -0,0 +1,1066 @@
+- Included license issue warning badges on Admin and sidebar menu items  <!-- 8074 -->
+- Made error handling and feedback messages more consistent and actionable throughout the UI  <!-- 7952 -->
+- In the UI, we renamed all `/cluster/*` routes to `/admin/*`  <!-- 8002 -->
+- Updated CLI version in the backend to 2.27.1  <!-- 8089 -->
```
```suggestion
- Updated CLI version in the backend to 2.27.2  <!-- 8089 -->
```
